src/datamodel/quests/quest_new.rs

```diff
@@ -0,0 +1,71 @@
+use std::collections::HashMap;
+use std::rc::Rc;
+
+#[derive(Debug, PartialEq)]
+pub enum Vote { Success, Failed, }
+
+pub struct QuestObjective {
+    pub num_players: u32,
+    pub required_successes: u32,
+}
+
+pub struct QuestNew {}
+
+impl QuestNew {
+    pub fn new(members: Vec<&'static str>, objective: Rc<QuestObjective>) -> OnGoingQuest {
```

Comment:
Clippy indicates that `new` methods usually return self. Maybe we can use this method inside of `OnGoingQuest` since the latter is the initial state for quests. Please tell me in case that I'm missing the point.

Reply:
In the Strategy or State Pattern constructors might return a different state type from the own. As you can see in this section of the book https://doc.rust-lang.org/book/ch17-03-oo-design-patterns.html#encoding-states-and-behavior-as-types

Reply:
I agree that there's no need for this struct if it's not doing anything except having a `new` that returns an instance from another struct...

Reply:
Indeed, let's merge OnGoingQuest and QuestNew then